Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -180,6 +180,16 @@ WK_SYSTEM_CONFIGURATION_LDFLAGS_macosx = -framework SystemConfiguration;
 
 WK_CORE_IMAGE_LDFLAGS = -framework CoreImage;
 
+WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS = $(WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS_$(WK_PLATFORM_NAME));
+WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS_iphoneos = $(WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS$(WK_IOS_14));
+WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS_iphonesimulator = $(WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS$(WK_IOS_14));
+WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS_maccatalyst = $(WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS$(WK_IOS_14));
+WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS_IOS_SINCE_14 = -framework UniformTypeIdentifiers;
```

Comment:
Is this necessary? I certainly don't think trunk builds for iOS 13 anymore.

Reply:
Fixed!